src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapTheNewGeneration.java

```diff
@@ -99,132 +99,132 @@ public void vuapDefaultStateAutoplayIsMutredAfrerReplayIsNotMuted(Page page, Str
+//    AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
+//    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+//    ads.refreshPage();
+//    scrollToSlot(slot, ads);
+//
+//    vuap.replay();
+//
+//    VuapAssertions.verifyVideoPlay(vuap);
+//  }
+//
+//  @Test(groups = {"AdsVuapResolvedStateTng", "AdsVuapResolvedStateTimeProgressOasis"},
+//      dataProviderClass = AdsDataProvider.class,
+//      dataProvider = "adsVuapTngDesktop")
+//  public void vuapResolvedStateShouldProgressInTime(Page page, String slot, String videoIframeSelector) {
+//    AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
+//    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+//    ads.refreshPage();
+//    scrollToSlot(slot, ads);
+//
+//    vuap.replay();
+//
+//    VuapAssertions.verifyVideoTimeIsProgressing(vuap);
+//  }
+//
+//  @Test(groups = {"AdsVuapResolvedStateTng", "AdsVuapResolvedStateClickOasis"},
+//      dataProviderClass = AdsDataProvider.class,
+//      dataProvider = "adsVuapTngDesktop")
+//  public void vuapResolvedStateShouldHaveLinkToProject43ArticleOnImage(Page page, String slot, String videoIframeSelector) {
+//    final AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
+//    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+//    ads.refreshPage();
+//    scrollToSlot(slot, ads);
+//
+//    vuap.clickOnClickArea2();
+//
+//    final String actual = ads.switchToNewBrowserTab();
+//    Assert.assertTrue(actual.equals(PROJECT43_TNG_ARTICLE_LINK), "Image should point to page on project43.");
+//  }
+//
+//  @Test(groups = {"AdsVuapResolvedStateTng", "AdsVuapResolvedStateClickOasis"},
+//      dataProviderClass = AdsDataProvider.class,
+//      dataProvider = "adsVuapTngDesktop")
+//  public void vuapResolvedStateShouldHaveLinkToFandomArticleOnImage(Page page, String slot, String videoIframeSelector) {
+//    final AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
+//    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+//    ads.refreshPage();
+//    scrollToSlot(slot, ads);
+//
+//    vuap.clickOnClickArea4();
+//
+//    final String actual = ads.switchToNewBrowserTab();
+//    Assert.assertTrue(actual.equals(FANDOM_ARTICLE_WESTWORLD_LINK), "Image should point to page on fandom.");
+//  }
+//
+//  @Test(groups = {"AdsVuapResolvedStateTng", "AdsVuapResolvedStateCloseOasis"},
+//      dataProviderClass = AdsDataProvider.class,
+//      dataProvider = "adsVuapTngDesktop")
+//  public void vuapResolvedStateShouldCloseAfterTapingOnCloseButton(Page page, String slot, String videoIframeSelector) {
+//    final AdsBaseObject ads = openPageWithVideoInLocalStorage(page);
+//    final AutoplayVuap vuap = new AutoplayVuap(driver, slot, videoIframeSelector);
+//    ads.refreshPage();
+//    scrollToSlot(slot, ads);
+//
+//    vuap.replay();
+//
+//    vuap.pause();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)